Source/WebCore/style/values/borders/StyleCornerShapeValue.h

```diff
@@ -40,26 +40,27 @@ using SuperellipseFunction = FunctionNotation<CSSValueSuperellipse, Number<CSS::
 struct CornerShapeValue {
     SuperellipseFunction superellipse;
 
-    static constexpr CornerShapeValue round() { return { SuperellipseFunction { 2.0 } }; }
-    static constexpr CornerShapeValue scoop() { return { SuperellipseFunction { 0.5 } }; }
-    static constexpr CornerShapeValue bevel() { return { SuperellipseFunction { 1.0 } }; }
-    static constexpr CornerShapeValue notch() { return { SuperellipseFunction { 0.0 } }; }
-    static constexpr CornerShapeValue straight() { return { SuperellipseFunction { std::numeric_limits<double>::infinity() } }; }
-    static constexpr CornerShapeValue squircle() { return { SuperellipseFunction { 4.0 } }; }
+    constexpr CornerShapeValue(CSS::Keyword::Round) : superellipse { 2.0 } { }
+    constexpr CornerShapeValue(CSS::Keyword::Scoop) : superellipse { 0.5 } { }
+    constexpr CornerShapeValue(CSS::Keyword::Bevel) : superellipse { 1.0 } { }
+    constexpr CornerShapeValue(CSS::Keyword::Notch) : superellipse { 0.0 } { }
+    constexpr CornerShapeValue(CSS::Keyword::Straight) : superellipse { std::numeric_limits<double>::infinity() } { }
+    constexpr CornerShapeValue(CSS::Keyword::Squircle) : superellipse { 4.0 } { }
+    constexpr CornerShapeValue(SuperellipseFunction value) : superellipse { value } { }
```

Comment:
Should this use `std::move` or `WTFMove`?

Reply:
Nah, Superellispe is just a thin wrapper around a double.